test/crawler/keyword/keywords_test.exs

```
@@ -17,6 +17,46 @@ defmodule Crawler.Keyword.KeywordsTest do
     end
   end
 
+  describe "get_keyword_by_id/1" do
+    test "given a valid keyword id, returns stored keyword" do
+      %{id: user_id} = insert(:user)
+      keyword = insert(:keyword, user_id: user_id, name: "keyword")
+
+      assert stored_keyword = Keywords.get_keyword_by_id(keyword.id)
+      assert stored_keyword.id == keyword.id
+      assert stored_keyword.name == "keyword"
+      assert stored_keyword.user_id == user_id
+    end
+
+    test "given an invalid keyword id, returns nil" do
+      %{id: user_id} = insert(:user)
+      insert(:keyword, user_id: user_id, name: "keyword")
+
+      assert Keywords.get_keyword_by_id(-1) == nil
+    end
+  end
+
+  describe "get_keyword_by_user_id_and_id/2" do
+    test "given a valid keyword id, returns stored keyword" do
+      %{id: user_id} = insert(:user)
+      keyword = insert(:keyword, user_id: user_id, name: "keyword")
+
+      assert stored_keyword = Keywords.get_keyword_by_user_id_and_id(user_id, keyword.id)
+      assert stored_keyword.id == keyword.id
+      assert stored_keyword.name == "keyword"
+      assert stored_keyword.user_id == user_id
+    end
+
+    test "given a keyword belongs to another user, returns nil" do
```

as comment above, let it raise the exception :)